cmd/cosign/cli/sign/sign.go

```diff
@@ -418,19 +426,11 @@ func SignerFromKeyOpts(ctx context.Context, certPath string, ko KeyOpts) (*PKCS1
 		}
 	}
 
-	var k *fulcio.Signer
-
-	if ko.InsecureSkipFulcioVerify {
-		k, err = fulcio.NewSigner(ctx, tok, ko.OIDCIssuer, ko.OIDCClientID, fClient)
-		if err != nil {
-			return nil, errors.Wrap(err, "getting key from Fulcio")
-		}
```

Comment:
I kinda think this commit makes things worse 🤣

Reply:
yeah shrug